gossipd/gossip.c

```diff
@@ -1627,6 +1629,32 @@ static struct io_plan *conn_init(struct io_conn *conn, struct reaching *reach)
 	return io_connect(conn, &ai, connection_out, reach);
 }
 
+static struct addrhint *
+seed_resolve_addr(const tal_t *ctx, const struct pubkey *id, const u16 port)
+{
+	struct addrhint *a;
+	char bech32[100], *addr;
+	u8 der[PUBKEY_DER_LEN];
+	u5 *data = tal_arr(ctx, u5, 0);
+
+	pubkey_to_der(der, id);
+	bech32_push_bits(&data, der, PUBKEY_DER_LEN*8);
+	bech32_encode(bech32, "ln", data, tal_count(data), sizeof(bech32));
+	addr = tal_fmt(ctx, "%s.lseed.bitcoinstats.com", bech32);
+
+	status_trace("Resolving %s", addr);
+
+	a = tal(ctx, struct addrhint);
+	if (!wireaddr_from_hostname(&a->addr, addr, 9735, NULL)) {
```

Comment:
This function `seed_resolve_addr` has `port` as argument, should it not propagate that from caller instead of hardcoding 9735 here?